src/core/symbology/qgsmarkersymbollayer.h

```diff
@@ -74,6 +74,9 @@ class CORE_EXPORT QgsSimpleMarkerSymbolLayerBase : public QgsMarkerSymbolLayer
       Octagon, //!< Octagon (since QGIS 3.18)
       SquareWithCorners, //!< A square with diagonal corners (since QGIS 3.18)
       AsteriskFill, //!< A filled asterisk shape (since QGIS 3.18)
+      SemiArc, //!< A line-only semi arc (since QGIS 3.20)
```

Comment:
Should this be "half arc" for clarity?

Reply:
Yeah maybe -- you want only the description changed or the shape enum / string too?

Reply:
Probably best to do it all....